### ISSUE TITLE
Improving image view counter / Исправить счетчик просмотров

### ISSUE THREAD

**Issue by Duche3d** [PastVu/pastvu#38]
Improving incorrect behavior of the image view counter: currently this is considered any refresh as the view.
To be: Only unique visitors per day should be considered as views.

Ru: Счетчик просмотров фото работает некорректно и считает за просмотр любой рефреш

![image](https://user-images.githubusercontent.com/60163100/73749625-caa3e480-476c-11ea-91d9-5b83f362309a.png)

**Comment by kabalin** [comment]
Тут не совсем "исправить", нужен способ хранения просмотренных фотографий. Простой способ - в local storage браузера хранить массив просмотренных фото в формате `cid` > `timestamp`. При просмотре фото, проверять есть ли cid текущего снимка в массиве и сколько времени прошло с последнего просмотра, если просмотр учитывать не нужно - передавать серверу в запросе соотвествующий флаг. Решение можно принимать и на стороне сервера если вместо флага передавать `timestamp`. Более сложный вариант - хранить массив просмотров для пользователя/сессии на стороне сервера и учитывать в подсчете (это кстати позволит показывать список пользователей кто посмотрел фото, если эта информация может быть полезной).

> Only unique visitors per day should be considered as views.

Стоит ли считать новым просмотром если один и тот же "браузер" посмотрел тот же снимок через день? Может быть корректнее считать уникальные просмотры за все время?

**Comment by paul-k-pastvu** [comment]
Да, желательно считать уникальные просмотры за все время - примерно так, как эту метрику учитывают другие сервисы.
Уникальные просмотры по дням - отдельная интересная статистика, если получится её сделать без больших доп. трудозатрат.
